dist/stat_data.py

```diff
@@ -537,11 +537,6 @@ def __init__(self, name, desc, flags=''):
     TxnStat('txn_checkpoint_time_recent', 'transaction checkpoint most recent time (msecs)', 'no_clear,no_scale'),
     TxnStat('txn_checkpoint_time_total', 'transaction checkpoint total time (msecs)', 'no_clear,no_scale'),
     TxnStat('txn_commit', 'transactions committed'),
-    TxnStat('txn_durable_queue_empty', 'durable timestamp queue insert to empty'),
-    TxnStat('txn_durable_queue_head', 'durable timestamp queue inserts to head'),
-    TxnStat('txn_durable_queue_inserts', 'durable timestamp queue inserts total'),
-    TxnStat('txn_durable_queue_len', 'durable timestamp queue length'),
-    TxnStat('txn_durable_queue_walked', 'durable timestamp queue entries walked'),
```

Comment:
Not sure if this is the right place to ask this question, but I'm wondering if we should add metrics to track the performance of the replacement mechanism. These metrics have been very useful in identifying this performance issue, and would think we want comparable metric for the new mechanism. For example, regarding "reader scanning all the sessions to get the min/max read and durable timestamps" this sounds to me like a potential area of performance concern (there can be a very large number of sessions, I believe) - should we have a metric to track the number of sessions scanned?

Reply:
We are already walking through all the sessions while forming the transaction snapshot. This approach adds in other flows also to walk through all the sessions. Yes, I agree that this approach can be a potential problem for some workloads. I added a new stat `transaction walk of concurrent sessions` to find out how many times this walk happened.

Reply:
Little fuzzy on the details here, but, if it makes sense, is that number of walks of all sessions, or total number of sessions walked? Both seem potentially useful, but the latter is maybe most directly related to performance?

Reply:
It is the number of times the sessions are walked. There is another stat that indicates the number of open sessions, this may not be equal to the number of active sessions, but this will be a close number due to which I didn't add another stat to indicate how many active sessions are there.